packages/react/src/components/TextField/index.tsx

```diff
@@ -80,10 +85,14 @@ export default class TextField extends React.Component<
     // textarea and input props being incompatible
     // we should probably fix this
     const Field: any = multiline ? 'textarea' : 'input';
+    const ariaDescribedbyWithDescription = description
```

Comment:
Just a very small nit. The naming of this implies it comes with a description, which isn't always the case since the description is conditional. I think we can just make this more generic.
```suggestion
    const ariaDescribedby = description
```

Reply:
The variable with such name already exists, but I share your view, so I renamed the target variable